storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt

```diff
@@ -0,0 +1,21 @@
+package com.icerockdev.service.storage.preview
+
+import com.sksamuel.scrimage.ScaleMethod
+
+enum class PreviewScaleMethod {
+    FastScale,
+    Lanczos3,
+    BSpline,
+    Bilinear,
+    Bicubic;
+
+    fun getScale(): ScaleMethod {
```

Comment:
internal please. without internal you can get same problem as you fix in `PngPreviewImpl`

Reply:
Fixed